### PR TITLE
 Refactor team page

### PR DESCRIPTION
**Pull request by Ana06**
To avoid having repeated code and make easy updating the information and add/remove members. :bowtie:

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit 655bdf35a90ba56ca9fc430afc3a14a662fa3eef

https://deploy-preview-156--openbuildservice.netlify.com

File: team/index.html [hunk 1]
```diff
@@ -1,403 +1,189 @@
...
-          <i class="big comments outline icon link" title="mdeniz at IRC (Freenode)" data-content="mdeniz at IRC (Freenode)"></i>
-        </div>
-      </div>
+  - name: Christian Bruckmayer
+    position: Frontend Developer
+    email: cbruckmayer@suse.com
+    github: chrisbr
+    twitter: bruckmayer
+    irc: ChrisBr
+    description: "After Marcus the second pup of the OBS team! Christian just _recently_ finished university and got into working full time on the OBS.
```
Comment (Ana06) [inline team/index.html]: @ChrisBr recently? :stuck_out_tongue_winking_eye:

Comment (ChrisBr) [reply team/index.html]: Time flies... I should update this sometime!

File: team/index.html [hunk 2]
```diff
@@ -1,403 +1,189 @@
...
-      </div>
+    \nWhen she is not hacking you may find her painting, cooking or learning some German."
+    
+  - name: Eduardo Navarro
+    position: Frontend Developer
+    email: enavarro@suse.de
+    github: eduardoj
+    twitter: eduardoj
+    irc: enavarro
+    description: "Used to be a happy Perl programmer living at Gran Canaria.
```
Comment (ChrisBr) [inline team/index.html]: Maybe you should add that Eduardo is our _Consultant_ :laughing:

File: team/index.html [hunk 3]
```diff
@@ -1,403 +1,189 @@
...
-            <span class="date">Project Manager</span>
-          </div>
-        </div>
-        <div class="extra content">
-          <a href="mailto:adrian@suse.de" target="_blank" title="Email Adrian"><i class="big mail icon"></i></a>
-          <a href="http://github.com/adrianschroeter" target="_blank" title="Adrian on Github"><i class="big github icon"></i></a>
-          <i class="big comments outline icon link" title="adrianS at IRC (Freenode)" data-content="adrianS at IRC (Freenode)"></i>
-        </div>
-      </div>
+team_members:
```
Comment (ChrisBr) [inline team/index.html]: Why not moving this to a dedicated file?

File: team/index.html (hunk 2 above)
Comment (Ana06) [reply team/index.html]: @eduardoj updated his linkedIn yesterday :rofl:

File: team/index.html (hunk 3 above)
Comment (ChrisBr) [reply team/index.html]: I don't really like to have it inside the "html"

Comment (Ana06) [reply team/index.html]: because that is how Jekyll works: https://jekyllrb.com/docs/frontmatter/

or how do you want do move it? :confused:

Comment (Ana06) [reply team/index.html]: but I think it makes sense to put it here... it is the team members page, it makes sense that the members definition is there.

Also, moving it to the config has some problems, as you need to restart the server to try the changes.

**Pull request merged**